### PR TITLE
Fix variable reference and exit status

### PR DESCRIPTION
**Pull request by ljl1988com**

### CODE REVIEW

File: bin/alluxio [hunk 1]
```diff
@@ -223,11 +223,11 @@ function main {
           shift # remove -s param
         fi
       else
-        echo "${Usage} $0 format [-s]"
+        echo "Usage $0 format [-s]"
```
Comment (LuQQiu) [inline bin/alluxio:226]: Can you help change to to the format similar to 
```
    echo "Usage: alluxio killAll <WORD>"
```

File: bin/alluxio [hunk 2]
```diff
@@ -223,11 +223,11 @@ function main {
...
         exit 2
       fi
     elif [[ $# -gt 1 ]]; then
-      echo "${Usage} $0 format [-s]"
```
Comment (LuQQiu) [inline bin/alluxio:230]: Same here

**Review by LuQQiu — CHANGES_REQUESTED** [review]
Thanks for fixing the bug in the bash script! Left some minor comments, otherwise LGTM

File: bin/alluxio [hunk 3]
```diff
@@ -81,7 +81,7 @@ EOF
 function killAll {
   if [[ $# -ne 1 ]]; then
     echo "Usage: alluxio killAll <WORD>"
-    exit
+    exit 1
```
Comment (ljl1988com) [inline bin/alluxio:84]: https://github.com/Alluxio/alluxio/blob/837a41c68b956ded56ae0fd41f0bf26b52ffbf49/bin/alluxio#L112
but
https://github.com/Alluxio/alluxio/blob/837a41c68b956ded56ae0fd41f0bf26b52ffbf49/bin/alluxio#L227
and in file `integration/emr/alluxio-emr.sh` all exit 2.
Which one is better?@LuQQiu

Comment (apc999) [reply bin/alluxio:84]: perhaps exit 2 is better here
according to https://tldp.org/LDP/abs/html/exitcodes.html

**Comment by apc999** [comment]
thanks for our contribution @ljl1988com  . One minor comment left.

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks for improving the shell error message and return code!

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**